handsontable/src/plugins/dragToScroll/dragToScroll.js

```diff
@@ -97,9 +97,15 @@ export class DragToScroll extends BasePlugin {
   /**
    * Sets the value of the visible element.
    *
-   * @param {DOMRect} boundaries An object with coordinates compatible with DOMRect.
+   * @param {DOMRect|{left: number, right: number, top: number, bottom: number}} [boundaries] An object with
+   * coordinates. Contains the window boundaries by default. The object is compatible with DOMRect.
    */
-  setBoundaries(boundaries) {
+  setBoundaries(boundaries = {
```

Comment:
Can I ask to update the types as well? https://github.com/handsontable/handsontable/blob/feature/dev-issue-49/handsontable/types/plugins/dragToScroll/dragToScroll.d.ts#L12